sdk/search/arm-search/recordings/node/search_test/recording_querykeys_create_test.json

```diff
@@ -37,7 +37,7 @@
       },
       "ResponseBody": {
         "name": "testjskey",
-        "key": "Ry6d14Q7F6Isa9JIVYeDm9lWkUJfJ5Ga8UqH2RVSyIAzSeCOCJh3"
```

Comment:
How can we avoid this to be merged in the future?

Reply:
Let's use [sanitizers](https://github.com/Azure/azure-sdk-for-js/blob/main/sdk/test-utils/recorder/README.md#sanitizers) in the tests instead of manual replacements in the recordings.

Reply:
If we can use a sanitizer, I agree that will help protect future issues.

Reply:
Sure I'll help on sanitizers.